diagnostic.js

```diff
@@ -33,14 +41,22 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update(
+  {name: "Stark"}, {$set: {members:["Ned Stark", "Arya Stark", "Sansa Stark"]}},
+  {name: "Targaryen"},  {$set: {members: ["Viserys Targaryen", "Daenerys Targaryen"]}},
+  {name: "Arryn"},  {$set: {members:["Jon Arryn"]}},
+  {name: "Targaryen"}, {$push: {members: "Jon Snow"}}
+);
+
+
 // your code ends here
 
 
 // Question 3
 //
 // House Arryn is not honorable!  Delete their motto.
 
-// your code begins here
+db.houses.delete({motto: "As High as Honor"});
```

Comment:
This was a bit of a trick question. Deleting a property from an object is actually an `update` action, like this:

```js
db.houses.update({ name: "House Arryn"}, { name: "House Arryn"});
```

With `update`, the 1st argument is the thing we are searching by. If you are not using a method like $set or $push in the 2nd argument, that object will simply get replaced by the 2nd argument.